app/helpers/interviews_helper.rb

```diff
@@ -0,0 +1,8 @@
+module InterviewsHelper
+  def email_user(user_id)
+    users=User.all
+    email=''
+    users.each { |user| email=user.email if user.id==user_id }
+    user_email=email
+  end
+end
```

Comment:
@vvs79 
There are 2 things about this helper:
1. It is totally not needed.
2. It is really very and very bad.

Please explain both statements.